index.css

```diff
@@ -83,6 +159,18 @@ small {
   font-family: 'Dejavu Sans Mono', 'Courier New', 'Courier', monospace;
 }
 
+#example > p {
+  margin: auto;
+  width: min-content;
+  background-color: rgb(0, 0, 0);
```

Comment:
because most terminals are black by default. this is **only** important, if "dim" is checked i.e. the element has the class style-dim.
shouldn't be  neccessary with your [fix](https://github.com/Scriptim/bash-prompt-generator/pull/1#discussion_r410949678).